configure.d/config_os_misc4

```diff
@@ -369,6 +369,56 @@ if test x$ac_cv_have_ip_pktinfo = xyes; then
     AC_DEFINE(HAVE_IP_PKTINFO, 1, [Set if IP_PKTINFO is usable])
 fi
 
+#   Check whether IPV6_RECVPKTINFO is usable.
+#
+
+AC_CACHE_CHECK([for IPV6_RECVPKTINFO ],
+                [ac_cv_have_ipv6_recvpktinfo],
+[AC_LINK_IFELSE([AC_LANG_PROGRAM([
```

Comment:
Since this program is compiled and linked but not run, please replace it with the following:
```
return IPV6_RECVPKTINFO
```

Reply:
Updated, thank you.